app/models/banner.rb

```diff
@@ -0,0 +1,27 @@
+class Banner < ActiveRecord::Base
+  enum record_type: { "category": 0, "item": 1 }
```

Comment:
you can define an const, don't have to put record_type in db, because it's really like bannerable_type